app/styles/neuron/contents.scss

```diff
@@ -11,14 +11,56 @@
   }
 }
 
+.frame-background{
+  background: $black;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+  position: absolute;
+  opacity: 0.7;
+}
 .content-selected {
-  -webkit-box-shadow: 1px 1px 5px 2px rgba(204,204,204,1);
-  -moz-box-shadow: 1px 1px 5px 2px rgba(204,204,204,1);
-  box-shadow: 1px 1px 5px 2px rgba(244,252,184,1);
+  -webkit-box-shadow: 1px 1px 0px 0px $borderContent,
+                      2px 2px 0px 0px $borderContent,
+                      3px 3px 0px 0px $borderContent,
+                      4px 4px 0px 0px $borderContent,
+                      5px 5px 0px 0px $borderContent,
+                      6px 6px 0px 0px $borderContent,
+                      7px 7px 0px 0px $borderContent,
+                      8px 8px 0px 0px $borderContent;
+
+  -moz-box-shadow:  1px 1px 0px 0px $borderContent,
+                    2px 2px 0px 0px $borderContent,
+                    3px 3px 0px 0px $borderContent,
+                    4px 4px 0px 0px $borderContent,
+                    5px 5px 0px 0px $borderContent,
+                    6px 6px 0px 0px $borderContent,
+                    7px 7px 0px 0px $borderContent,
+                    8px 8px 0px 0px $borderContent;
+
```

Comment:
es para un efecto 3D que quiere @enactogen mira http://stackoverflow.com/questions/12660450/css-border-radius-3d-straight-effect-and-not-curved-effect